agent/src/test/java/com/spotify/ffwd/FfwdConfigurationTest.java

```diff
@@ -89,11 +89,15 @@ public void testConfMixedPluginsEnabled() {
     public void testConfigFromEnvVars() {
         environmentVariables.set("FFWD_TTL", "100");
         environmentVariables.set("FFWD_OUTPUT_RATELIMIT", "1000");
+        environmentVariables.set("FFWD_OUTPUT_CARDINALITYLIMIT", "10000");
+        environmentVariables.set("FFWD_OUTPUT_HYPERLOGLOGSWAPPERIODMS", "3000000");
```

Comment:
i'm having a hard time reading `HYPERLOGLOGSWAPPERIODMS` can that be made more readable?

Reply:
done